requirements/frozen_docs.py2.txt

```diff
@@ -172,6 +174,7 @@ Werkzeug==0.14.1
 widgetastic.core==0.21.3
 widgetastic.patternfly==0.0.34
 widgetsnbextension==3.2.1
+wrapanapi==3.0.14
```

Comment:
I had just removed wrapanapi from docs, because its all mocked. this was causing the doc env build to fail.